package.json

```diff
@@ -141,6 +143,7 @@
     "jeet": "*",
     "jquery": "*",
     "jquery.finger": "*",
+    "lodash": "^4.17.20",
```

Comment:
I'd remove this and just use the native method

Reply:
(Remember we don't have bundle splitting yet)

Reply:
Second @dzucconi on that, but also note that https://github.com/lodash/babel-plugin-lodash will pluck things out if you just _have to_ lol